force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -34,18 +34,17 @@ public with sharing class DummyJSONCallout {
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
 
+	@future(callout = true)
 	public static void getDummyJSONUserFromId(String dummyUserId) {
-		// Create HTTP request to send.
-
-		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
-		// Set the HTTP method to GET.
-
-		// Send the HTTP request and get the response.
-
-		// If the HTTP response code is successful, parse the JSON response and update the contact.
-		Contact cont = null;
-		upsert cont DummyJSON_Id__c; //insert/update from the JSON response using the external id (dummyUserId)
+		Http http = new Http();
+		HttpRequest request = new HttpRequest();
+		request.setEndpoint('https://dummyjson.com/user/' + dummyUserId);
+		request.setMethod('GET');
+		HttpResponse response = http.send(request);
+		if (response.getStatusCode() >= 200 || response.getStatusCode() <= 299) {
```

Comment:
```suggestion
		if (response.getStatusCode() >= 200 && response.getStatusCode() <= 299) {
```
This should be AND not OR. You want a code in the 200 range for success. This works because 200 is normally what is returned and you are using or. This would be true if the returned value was 400, which you would not want.